instructions for adding sample data to the infra views

vsingh-msys opened the pull request:
### :nut_and_bolt: Description: What code changed, and why?

- Infra views feature has been added recently, so it would nice to populate sample data.
- Add hab studio script to populate the dummy data for the infra servers & orgs.
- In order to fetch dummy data for the cookbooks, roles, etc required mocking which we will decide further.

<!-- /!\ Please ensure that you are NOT disclosing any customer information without their consent /!\ -->

### :chains: Related Resources
[MSYS-1251](https://chefio.atlassian.net/browse/MSYS-1251)
### :+1: Definition of Done
Ease to add sample data for the infra views testing.

### :athletic_shoe: How to Build and Test the Change
 - From hab studio `source .studio/infra-proxy-service` file to load the script.
-  Start `infra-proxy-service` service by running `start_infra_proxy_service` or `start_all_services`
 - Hit `infra_service_load_sample_data` will populate 50 records by default.
 - For more detail ref `dev-docs/adding-data/adding_test_data.md`

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable

Signed-off-by: Vivek Singh <vivek.singh@msystechnologies.com>

vsingh-msys commented:
Logs:
```
[18][default:/src:0]# infra_service_load_sample_data
  hab-studio: Started Loading sample data...
  hab-studio: Total number of records: 50
  hab-studio: Chef server name prefix: chef-server
  hab-studio: Chef server org name prefix: chef-org
  hab-studio: Sample data loaded...
[19][default:/src:0]# 
```

susanev commented on dev-docs/adding-data/adding_test_data.md:
```diff
@@ -66,6 +66,18 @@ _Note: to add an ec2 integration for aws, change type from "aws-api" to "aws-ec2
+option one: `infra_service_load_sample_data`
+By default, 50 records of the infra servers and orgs will be added with the prefix `chef-server` and `chef-org` respectively.
+
+option one: `infra_service_load_sample_data -N 100 -S infra-server -O infra-org`
```
```suggestion
option two: `infra_service_load_sample_data -N 100 -S infra-server -O infra-org`
```

susanev commented:
i tried to test this out, but i already had a all services running, and it wouldnt let me source because the deployment service was already running. i tried, starting a new studio and couldnt get it to get past the local manifest file step.... maybe im doing something wrong?

vsingh-msys commented:
@susanev seems that due to document enclose with ` which causes string interpolation and command `start_infra_proxy_service` get executed. Added fix and updated the PR.
Could you please have a look again and let me know if anything required?
Thanks!
